specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-07-01/examples/Accounts_CreateOrUpdateAD.json

```
@@ -0,0 +1,81 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "resourceGroupName": "myRG",
+    "accountName": "account1",
+    "api-version": "2023-07-01",
+    "body": {
+      "location": "eastus",
+      "properties": {
+        "activeDirectories": [
+          {
+            "site": "SiteName",
+            "username": "ad_user_name",
+            "password": "ad_password",
+            "domain": "10.10.10.3",
+            "dns": "10.10.10.3, 10.10.10.4",
```

```suggestion
            "dns": "10.10.10.3",
```
please update to this